README.md

```diff
@@ -22,12 +22,16 @@ To activate it in your Jest environment you have to include it in your configura
 }
 ```
 
-### Without Jest
+### With Playwright test runner
 
-```javascript
-import expect from "expect-playwright"
+To activate with the Playwright test runner, use `expect.extend` to add the `expect-playwright` matchers.
 
-await expect(page).toHaveText("#foo", "my text")
+```js
+// folio.config.ts
+import { expect } from "@playwright/test"
+import matchers from "expect-playwright"
+
+expect.extend(matchers)
```

Comment:
@mxschmitt I updated the readme with a sample of what the docs could look like.  I'm not real familiar with folio so this might very well need to be adjusted on how the next release will work.

Reply:
I would have waited with folio since there is a big breaking change incoming there soon, but we can also update it then.

Reply:
@mxschmitt I'm fine with waiting on the folio update before moving this forward.  That way it's one change, and it will continue to work for folio users in the mean time as is.  This PR isn't impeding any progress since it's just a quality of life change that doesn't impact functionality of the matchers themselves.

Do you have an estimate on when the folio breaking change will land so I can keep tabs and update this PR accordingly?

Reply:
This will probably take a few weeks and the users are afaik mostly using it with Jest. So I think we can merge the PR earlier.

Reply:
@mxschmitt Sounds good, in that case this is ready to go.  I did do one final change of exporting the matchers as a non-default export to be less confusing since the old default export was the custom `expect` wrapper.